src/guest_webapp/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+# not sure which version to use
+version: '3.8'
+
+services:
```

Comment:
It would be good to update the main docker compose file.
I request the following changes to the "docker-compose/demo-app.yaml":
1) Rename file to "docker-compose.yaml"
2) add the following to the top of that file:
```
include:
  - ../src/guest_webapp/docker-compose.yaml     # uses port 5173
  - ../src/staff_webapp/docker-compose.yaml     # uses port 4200
```